networking/understanding-networking.adoc

```diff
@@ -12,4 +12,10 @@ IP address means that Pods can be treated like physical hosts or virtual
 machines in terms of port allocation, networking, naming, service discovery,
 load balancing, application configuration, and migration.
 
+[NOTE]
+====
+Link-local IP addresses are assigned from the `169.254.0.0/16` CIDR block.
+For the CIDR block to be reachable by a Pod, the `spec.hostNetwork` field for the Pod CR must be set to `true`.
```

Comment:
I would say something like:

> Some cloud platforms provide APIs on addresses in the IPv4 link-local CIDR block (`169.254.0.0/16`). This CIDR block is not reachable from the pod network; Pods that need access to these IPs will need to be given host network access by setting the `spec.hostNetwork` field in the Pod spec to `true`.

Especially, it should be clearer that setting `spec.hostNetwork` is granting additional privileges to the pod (the wording I suggested may not be the best way to do that).

Also, "the Pod CR" is wrong. "CR" = "Custom Resource", which applies only types that are defined by Custom Resource Definitions. Built-in types like Pod are just "resources", not "custom resources".

Reply:
@danwinship, thanks. I've updated the text; How does it look now?